publish2cloud.py

```diff
@@ -49,15 +49,13 @@
     REMOTE_SETTINGS_AUTH = ''
     if os.environ.get('AUTHORIZATION', None):
         REMOTE_SETTINGS_AUTH = CONFIG.get('main', 'remote_settings_authorization')
-
     # We can remove the use of BearerAuth() once we switch to kinto-http
     if rs_auth_method == 'token':
         REMOTE_SETTINGS_AUTH = BearerAuth(REMOTE_SETTINGS_AUTH)
     elif rs_auth_method == 'userpass':
         REMOTE_SETTINGS_AUTH = HTTPBasicAuth(*tuple(REMOTE_SETTINGS_AUTH.split(":", maxsplit=1)))
 
     CLOUDFRONT_USER_ID = os.environ.get('CLOUDFRONT_USER_ID', None)
-    NUMBER_OF_SUPPORTED_VERSIONS = CONFIG.get('main', 'num_supported_versions')
```

Comment:
This variable was unused since I directly checked for the `num_supported_versions` variable from the config in `new_data_to_publish_remote_settings` function, which is only called when remote settings is enabled